bundles/Billing/Entity/Invoice.php

```diff
@@ -99,6 +99,11 @@ class Invoice
      */
     private $lockedSalesChannel;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $payedByAccount;
```

Comment:
“payed”?! Is this the nautical term (https://en.wiktionary.org/wiki/pay#Verb_2)?